client/softlayer_client.go

```diff
@@ -169,6 +178,15 @@ func (slc *SoftLayerClient) GetSoftLayer_Hardware_Service() (softlayer.SoftLayer
 	return slService.(softlayer.SoftLayer_Hardware_Service), nil
 }
 
+func (slc *SoftLayerClient) GetSoftLayer_Dns_Domain_Resource_Record_Service() (softlayer.SoftLayer_Dns_Domain_Resource_Record_Service, error) {
+	slService, err := slc.GetService("SoftLayer_Dns_Domain_ResourceRecord")
```

Comment:
Why is the method named: `GetSoftLayer_Dns_Domain_Resource_Record_Service` when the service is named `SoftLayer_Dns_Domain_ResourceRecord`? note the missing _ in the name? The convention I am using is to match the name with the SL name as in the API. So if the SL API use: "SoftLayer_Dns_Domain_ResourceRecord" then the service method would then be: `GetSoftLayer_Dns_Domain_ResourceRecord_Service` which is odd (missing the _) but that's consistent. Thoughts?